.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please add the file to gitignore:
https://www.atlassian.com/git/tutorials/saving-changes/gitignore